### PR TITLE
MODPATBLK-104 New lines removal from logs

### PR DESCRIPTION
**Pull request by bisecomsergiy**
### Purpose
Decrease quantity of logs lines produced by fetchById() method of the module
### Approach
Moved new lines available in json response body and structured data within one line
### Learning
https://issues.folio.org/browse/MODPATBLK-104

### CODE REVIEW

File: src/main/java/org/folio/rest/client/OkapiClient.java [hunk 1]
```diff
@@ -115,17 +115,17 @@ public OkapiClient(Vertx vertx, Map<String, String> okapiHeaders) {
...
         String errorMessage = String.format("Failed to fetch %s. Response: %d %s",
-          responseType.getName(), responseStatus, response.bodyAsString());
+          responseType.getName(), responseStatus, response.bodyAsString().replaceAll("\\r|\\n", ""));
         log.error(errorMessage);
         return failedFuture(new EntityNotFoundException(errorMessage));
       } else {
         try {
           T fetchedObject = objectMapper.readValue(response.bodyAsString(), responseType);
-          log.info("Fetched from {}. Response body: \n{}", path, response.bodyAsString());
+          log.info("Fetched from {}. Response body: {}", path, response.bodyAsString().replaceAll("\\r|\\n", ""));
```
Comment (roman-barannyk) [inline src/main/java/org/folio/rest/client/OkapiClient.java:124]: I would suggest moving it to constants

Comment (bisecomsergiy) [reply src/main/java/org/folio/rest/client/OkapiClient.java]: Thank you, it is done..

File: src/main/java/org/folio/rest/client/OkapiClient.java [hunk 2]
```diff
@@ -30,6 +30,8 @@
 import io.vertx.ext.web.client.WebClient;
 
 public class OkapiClient {
+  public static final String EMPTY_STRING = "";
+  public static final String NEW_LINES_STRING = "\\r|\\n";
```
Comment (roman-barannyk) [inline src/main/java/org/folio/rest/client/OkapiClient.java:34]:
```suggestion
  public static final String LINE_SEPARATOR = "\\r|\\n";
```

File: src/main/java/org/folio/rest/client/OkapiClient.java [hunk 3]
```diff
@@ -30,6 +30,8 @@
 import io.vertx.ext.web.client.WebClient;
 
 public class OkapiClient {
+  public static final String EMPTY_STRING = "";
```
Comment (roman-barannyk) [inline src/main/java/org/folio/rest/client/OkapiClient.java:33]: StringUtil.EMPTY_STRING

Comment (bisecomsergiy) [reply src/main/java/org/folio/rest/client/OkapiClient.java]: Ok, thanks, it is done

Comment (bisecomsergiy) [reply src/main/java/org/folio/rest/client/OkapiClient.java]: I have used org.apache.commons.lang3.StringUtils

File: src/main/java/org/folio/rest/client/OkapiClient.java (hunk 2 above)
Comment (bisecomsergiy) [reply src/main/java/org/folio/rest/client/OkapiClient.java]: It is done, thank you

File: src/main/java/org/folio/rest/client/OkapiClient.java [hunk 4]
```diff
@@ -72,17 +74,17 @@ public OkapiClient(Vertx vertx, Map<String, String> okapiHeaders) {
...
         String errorMessage = String.format("Failed to fetch %s by ID: %s. Response: %d %s",
-          responseType.getName(), id, responseStatus, response.bodyAsString());
+          responseType.getName(), id, responseStatus, response.bodyAsString().replaceAll(LINE_SEPARATOR, EMPTY));
         log.error(errorMessage);
         return failedFuture(new EntityNotFoundException(errorMessage));
       } else {
         try {
           T fetchedObject = objectMapper.readValue(response.bodyAsString(), responseType);
-          log.info("Fetched by ID: {}/{}. Response body: \n{}", path, id, response.bodyAsString());
+          log.info("Fetched by ID: {}/{}. Response body: {}", path, id, response.bodyAsString().replaceAll(LINE_SEPARATOR, EMPTY));
```
Comment (OleksandrVidinieiev) [inline src/main/java/org/folio/rest/client/OkapiClient.java:83]: Please change logging level for bodies of successful responses from `info` to `debug`.

Comment (bisecomsergiy) [reply src/main/java/org/folio/rest/client/OkapiClient.java]: It is done, thank you

**Review by alexanderkurash — APPROVED** [review]

**Pull request merged**